Add IFToMM 2018

octave-user opened the pull request:
Add link to [IFToMM 2018](https://www.researchgate.net/publication/327108298_Analysis_of_Analytical_Hydrodynamic_Bearing_Models_on_a_Reciprocating_Compressor_Vol_1)

octave-user committed:
Add IFToMM 2018

mmorandi commented on Documentation/Publications.md:
```diff
@@ -57,185 +57,186 @@ _If your work uses **MBDyn**, please cite these papers as appropriate_:
-* William T. Yeager Jr., Raymond G. Kvaternik "**[Contributions of the Langley Transonic Dynamics Tunnel to rotorcraft technology and development](http://www.cs.odu.edu/~mln/ltrs-pdfs/NASA-aiaa-2000-1771.pdf)**" AIAA-2000-1771 AIAA/ASME/ASCE/AHS/ASC Structures, Structural Dynamics, and Materials Conference and Exhibit, 41st, Atlanta, GA, Apr. 3-6, 2000. 
-* Thomas Hablowetz "**Advanced Helicopter Flight and Aeroelastic Simulation based on General Purpose Multibody Code**" presented at the _AIAA Modeling and Simulation Technologies Conference and Exhibit_ Denver, CO, USA, August 14-17, 2000 (AIAA 2000-4299) 
+## Conference Papers:
+* E. P. Okabe, J. H. Izuka, R. Resch ["**Analysis of Analytical Hydrodynamic Bearing Models on a Reciprocating Compressor**"](https://www.researchgate.net/publication/327108298_Analysis_of_Analytical_Hydrodynamic_Bearing_Models_on_a_Reciprocating_Compressor_Vol_1) 10th International Conference on Rotor Dynamics – [IFToMM 2018](https://link.springer.com/chapter/10.1007/978-3-319-99262-4_22), Rio d ...
```
If I get it right, this paper uses OpenModelica, and no mention is made about MBDyn. Why cite it here? Does the implementation/model matches your work in MBDyn (likely yes, but perhaps we can explain the citation)?

octave-user replied:
Well, Eduardo Okabe used the old Fortran 90 code from module-hydrodynamic_plain_bearing for his paper. Actually, that's the only reason why I'm one of authors. In the meantime, the old Fortran code was converted to C++. Nevertheless, it is still the same algorithm.

octave-user replied:
OK, I have added this remark and link:
![image](https://github.com/user-attachments/assets/4f1a123e-364a-4827-a0b2-648c1b9ca31c)

mmorandi replied:
Merged and published on www.mbdyn.org .
Thank you!

octave-user committed:
Merge branch 'main' into publications

octave-user committed:
Add remark and link

Eduardo Okabe used the old Fortran 90 code from module-hydrodynamic_plain_bearing for his paper. Actually, that's the only reason why I'm one of authors. In the meantime, the old Fortran code was converted to C++. Nevertheless, it is still the same algorithm.

octave-user committed:
Fix link to linkage
